Source/Core/Common/Thread.cpp

```diff
@@ -83,7 +83,7 @@ void SetCurrentThreadName(const char* szThreadName)
 
   info.dwType = 0x1000;
   info.szName = szThreadName;
-  info.dwThreadID = -1;  // dwThreadID;
+  info.dwThreadID = (DWORD)-1;
```

Comment:
I use -1 here because it is the documented value. It doesn't really matter to me, though.